app/controllers/application_controller/ci_processing.rb

```diff
@@ -1067,7 +1067,7 @@ def set_memory_cpu
     #set memory to nil if multiple items were selected with different mem_cpu values
     @reconfigureitems.each_with_index do |vm, i|
       @temp_memory = i == 0 || @temp_memory == vm.mem_cpu ? vm.mem_cpu : nil
-      @temp_cpu_count = i == 0 || @temp_cpu_count == vm.num_cpu ? vm.num_cpu : nil
+      @temp_cpu_count = i == 0 || @temp_cpu_count == vm.logical_cpus ? vm.logical_cpus : nil
```

Comment:
If we're going to attempt to improve this... why instance variables? :crying_cat_face: 

I imagine there's also a better overall spelling of the intent. I'm not thrilled with these, but they at least seem an improvement:
```ruby
memory = @reconfigureitems.map(&:mem_cpu).uniq.one? ? @reconfigureitems.first.mem_cpu : nil
# or
memory = @reconfigureitems.first.mem_cpu
memory = nil unless @reconfigureitems.all? { |vm| vm.mem_cpu == memory }
```